ODT/ODTUtility.php

```diff
@@ -198,7 +198,13 @@ public static function getImageSize($src, $maxwidth=NULL, $maxheight=NULL){
             $info = getimagesize($src);
         } else {
             // FIXME: Add cache support for downloaded images.
-            $fetch = @(new DokuHTTPClient())->get($src);
+            if (class_exists('dokuwiki\HTTP\DokuHTTPClient')) {
+                $http = new dokuwiki\HTTP\DokuHTTPClient();
+            } else {
+                $http = new DokuHTTPClient();
+            }
+            $fetch = @$http->get($src);
+            var_dump($fetch);
```

Comment:
Some debugging code is left..

Reply:
I know, this PR was removed and created a separate PR for this, as it's not related to this PR.